src/bin/conserve.rs

```diff
@@ -179,6 +179,21 @@ enum Command {
         long_listing: bool,
     },
 
+    /// Mount the archive as projection.
+    #[cfg(windows)]
+    Mount {
+        /// The archive to mount
+        archive: String,
+
+        /// Target folder where the archive should be mounted to
+        destination: PathBuf,
+
+        /// Create the target folder and remove all temporarily created
+        /// files on exit
```

Comment:
It depends on the use case. In general, I would recommend going with the clean-up option, which is why it's the default setting.

For a projection, you typically expect no files to be stored once stopped, as they are merely projected. However, the way Windows ProjFS works under the hood is that when a file is accessed, it is written locally, which is why the clean-up is necessary.

That being said, if you regularly access certain files, it might be useful to skip the clean-up and use the locally stored files as a basic "cache."

Reply:
I see, thanks.